lookout/style/format/quality_report.py

```diff
@@ -13,6 +13,27 @@
 from lookout.style.format.utils import prepare_files
 
 
+def generate_report(y, y_pred, vnodes_y, n_files, target_names):
+    report = list()
```

Comment:
Please use `io.StringIO` instead

Reply:
do you mean like this, @vmarkovtsev ?

```python
import io

output = io.StringIO()
output.write('First line.\n')
print('Second line.', file=output)

# Retrieve file contents -- this will be
# 'First line.\nSecond line.\n'
contents = output.getvalue()

# Close object and discard memory buffer --
# .getvalue() will now raise an exception.
output.close()
```

Reply:
Yes. Please use `with output` though.

Reply:
@EgorBu ping